english/pinch/jacket/eatit/eatit.md

```diff
@@ -1 +1 @@
-You eat a hole threw the Marshmallow wall. Looking through the hole you see a hallway.
\ No newline at end of file
+You eat a hole threw the Marshmallow wall. Looking through the [hole](hole/hole.md) you see a [hallway](hallway/hallway.md).
```

Comment:
This line contains more than one link.  Please put links on separate lines with a blank line in between so that options will be visually separated.